Test/dafny/dafny-issue-4804.bpl

```diff
@@ -0,0 +1,3567 @@
+// RUN: %parallel-boogie /proverOpt:O:auto_config=false /proverOpt:O:type_check=true /proverOpt:O:smt.qi.eager_threshold=100 /proverOpt:O:smt.delay_units=true /rlimit:10000 /prune "%s" > "%t"
```

Comment:
Triggering an rlimit threshold breach is easy. Triggering one that Z3 reports as "unknown" instead of something like "resource limits reached" is harder. I'm sure there is a smaller example that would trigger the issue, but I don't know what it is.